images/win/Windows2022-Readme.md

```diff
@@ -92,25 +93,20 @@
 - Hub CLI 2.14.2
 
 ### Rust Tools
-- Cargo 1.57.0
-- Rust 1.57.0
-- Rustdoc 1.57.0
+- Cargo 
+- Rust 
+- Rustdoc 
```

Comment:
@V-Zabayrachny something went wrong. Is rust not installed at all or there is an issue with the version retrieval?

Reply:
@miketimofeev Rust is installed, there is an issue with the version retrieval, this will be fixed as part of this PR https://github.com/actions/virtual-environments/pull/4774/files